runs.py

```diff
@@ -120,6 +121,46 @@ def removals(self) -> list[str]:
         all_removals = self.neow_bonus.cards_removed + event_removals + store_removals
         return all_removals
 
+    @property
+    def character_streak(self) -> StreakInfo:
+        """Gets the run position in the character streak."""
+        return self._get_streak(True)
+
+    @property
+    def rotating_streak(self) -> StreakInfo:
+        """Gets the run position in the rotating streak."""
+        return self._get_streak(False)
+
+    def _get_streak(self, is_character_streak: bool) -> StreakInfo:
```

Comment:
boolean arguments should be keyword-only (so the signature should be `self, *, is_character_streak: bool`), and the two properties should call by the arg name